lib/src/widgets/typing/ds_dot.widget.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class DotWidget extends StatelessWidget {
```

Comment:
Use the prefix 'DS' and without 'widget' in the class name, only in the file name

Reply:
Done